internal/executor/rpc/rpc.go

```diff
@@ -140,6 +140,9 @@ func (r *RPC) Start(ctx context.Context) error {
 
 	listener, err := net.Listen("tcp", address)
 	if err != nil {
+		if strings.Contains(err.Error(), "bind: cannot assign requested address") {
```

Comment:
I don't think this will work for non-english locales. A better way would be to check for `errors.Is(err, syscall.EADDRNOTAVAIL)`.

However, both won't probably work on Windows anyway.

Reply:
Seems so but at least on majority platforms the error message will be better.